app/src/main/kotlin/no/nav/k9punsj/omsorgspengermidlertidigalene/OmsorgspengerMidlertidigAleneService.kt

```diff
@@ -81,21 +80,24 @@ internal class OmsorgspengerMidlertidigAleneService(
 
     internal suspend fun nySøknad(request: ServerRequest, nyOmsMASøknad: NyOmsMASøknad): ServerResponse {
         // oppretter sak i k9-sak hvis det ikke finnes fra før
-        if(nyOmsMASøknad.annenPart != null) {
-            val hentK9SaksnummerGrunnlag = HentK9SaksnummerGrunnlag(
-                søknadstype = FagsakYtelseType.OMSORGSPENGER_MIDLERTIDIG_ALENE,
-                annenPart = nyOmsMASøknad.annenPart,
-                søker = nyOmsMASøknad.norskIdent,
-                pleietrengende = null,
-                periode = Periode.ÅpenPeriode
-            )
+        val hentK9SaksnummerGrunnlag = HentK9SaksnummerGrunnlag(
+            søknadstype = FagsakYtelseType.OMSORGSPENGER_MIDLERTIDIG_ALENE,
+            annenPart = nyOmsMASøknad.annenPart,
+            søker = nyOmsMASøknad.norskIdent,
+            pleietrengende = null,
```

Comment:
Hvordan håndteres liste med barn?

Reply:
Kun annenPart som brukes her for midlertidig alene.

Reply:
Flott!